playbooks/google.yml

```diff
@@ -121,9 +128,15 @@
       prompt: "\nStreisand will now set up your server. This process usually takes around ten minutes. Press Enter to begin setup...\n"
 
   pre_tasks:
-    - name: Set the Google Compute Engine Zone fact
+    - name: Set the Google Compute Engine Zone interactive
       set_fact:
-        gce_zone: "{{ zones[gce_zone_var] }}"
+        gce_zone: "{{ zones[gce_zone_idx] }}"
+      when: gce_zone is not defined
+
+    - name: Set the Google Compute Engine Zone noninteractive
```

Comment:
Hm, wait, do we need this clause at all? Directly above it, we've set gce_zone, so this clause should have the effect of `gce_zone := gce_zone`?

Do I need more coffee?